package.json

```diff
@@ -37,6 +37,6 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha -u tdd"
```

Comment:
When trying to run the test suite, I kept getting `ReferenceError: suite is not defined`. Adding this argument forces Mocha to use the TDD interface and it's able to execute the tests.

Reply:
Thank you. I think Mocha has either changed the location of the config file, or deprecated its use completely.